frontend/components/MainPageV1/Home/Overview/Performance.tsx

```diff
@@ -45,6 +45,9 @@ const MetricsCapturedTimestampAlert = ({
   />
 );
 
+const AGENT_BEHAVIOR_TEXT =
+  'Conservative volatile exposure across DEXs and lending markets with advanced functionality enabled.';
+
```

Comment:
we need to use default behaviour text which is different for each agents? see below
```suggestion
```